css/elements/commons.css

```diff
@@ -0,0 +1,17 @@
+a {
+    color: #151515;
+    text-decoration: auto;
+}
+
+span {
+    line-height: 26px;
+}
+
+button {
+    height: 52px;
+    cursor: pointer;
+}
+
+input {
+    height: 52px;
+}
```

Comment:
good naming, although I would change the use of `px` for `rem` and define a variable for that, maybe `--min-action-height` and then `block-size: var(--min-action-height)`

Reply:
Doing in next PR